Runtime/Core/ApiGatewayManagement/AmazonGameLiftWrapper.cs

```diff
@@ -42,6 +42,26 @@ public async Task<DescribeGameSessionsResponse> DescribeGameSessions(DescribeGam
         {
             return await _amazonGameLiftClient.DescribeGameSessionsAsync(request);
         }
+        
+        public async Task<ListFleetsResponse> ListFleets(ListFleetsRequest request)
+        {
+            return await _amazonGameLiftClient.ListFleetsAsync(request);
+        }
+        
+        public async Task<DescribeFleetAttributesResponse> DescribeFleets(DescribeFleetAttributesRequest request)
```

Comment:
This method should be called `DescribeFleetAttributes` to be consistent with the API naming

Reply:
renamed